build/Publish.targets

```diff
@@ -11,7 +11,6 @@
   <Target Name="Publish"
           Condition=" '$(PUBLISH_TO_AZURE_BLOB)' != '' "
           DependsOnTargets="Init;
-                            Package;
```

Comment:
Why is Package removed here?

Reply:
I swapped Package with "PackageAllLinux". So, if I do not remove it there, it will run the normal Package which I don't want, since I am reusing the package step.

I am not sure the best practice here. In a sense Publish does not require package (do nothing). The "normal" target, state explicit `compile, package, publish` so no behavior change